src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1342,6 +1278,27 @@ static bool processContextAttributeVector (ContextElement*
 
     }
 
+    /* Special processing in the case of DELETE, to avoid "deleting too much" in the case of using
+     * metadata ID. If metadata ID feature were removed this ugly piece of code wouldn't be needed ;)
+     * FIXME P3: note that in the case of Active/Active configuration it could happen that
+     * the attrsName get desyncrhonized, e.g. starting situation is A-ID1 and A-ID2, node
```

Comment:
typo: desynchronized   (not even sure whether that is a word ... :-)
Not important, it's self-explanatory.